cumulusci/cli/cci.py

```diff
@@ -34,11 +34,14 @@
 from cumulusci.core.config import TaskConfig
 from cumulusci.core.config import UniversalConfig
 from cumulusci.core.github import create_gist, get_github_api
-from cumulusci.core.exceptions import OrgNotFound
-from cumulusci.core.exceptions import CumulusCIException
-from cumulusci.core.exceptions import CumulusCIUsageError
-from cumulusci.core.exceptions import ServiceNotConfigured
-from cumulusci.core.exceptions import FlowNotFoundError
+from cumulusci.core.exceptions import (
+    OrgNotFound,
+    CumulusCIException,
+    CumulusCIUsageError,
+    ServiceNotConfigured,
+    FlowNotFoundError,
+)
```

Comment:
Please don't rewrite existing imports to group them like this. I'm fine with adding new imports in either style, but we don't have a standard preference for one over the other so I object to including it in the diff.

(Personally, I prefer the style you removed because it is easier to copy and paste between modules and helps produce diffs that only include the imports that changed.)

Reply:
Okay, I thought that the grouped style was the preference.